minesweeper.py

```diff
@@ -360,6 +362,14 @@ def changegrid(usery,userx):
 #gui stuff:
 root=Tk()
 frame=Frame(root)
+
+HEIGHT = 30
+WIDTH = 30
+path = "C:\\Users\\Tript\\Desktop\\What!\\bomb.jpg"

```

Comment:
This needs to be replaced with a relative path rather than an absolute path, in this case just changing it to "bomb.jpg" does the trick since it is in the same folder as the main python file

Reply:
Ah!! Forgot to do that. Wait for a few minutes check back again